pac4j-jwt/src/main/java/org/pac4j/jwt/profile/JwtProfile.java

```diff
@@ -0,0 +1,13 @@
+package org.pac4j.jwt.profile;
+
+import org.pac4j.core.profile.CommonProfile;
+
+/**
+ * Represents a user profile based on a JWT.
+ *
+ * @author Misagh Moayyed
+ * @since 1.8.2
+ */
+public class JwtProfile extends CommonProfile {
```

Comment:
I am for that idea. I don't see much difference between http and jwt profiles, which is why I thought originally to just return/use the http profile for jwt but I think it's best to keep the jwt profile separate and just have it extend http.